### ISSUE TITLE
[C2M] Option `-o` is ignored when using with `-s` or `-c`

### ISSUE THREAD

**Issue by edubart** [vnmakarov/mir#209]
When I do the following command:
```
c2m -S -o t.mir test.c
c2m -c -o t.bmir test.c
````

I was expecting that the files `t.mir` and `t.bmir` would be created, but instead,
the files `test.mir` and `test.bmir` where created, seems like `-o` is ignored.

**Comment by vnmakarov** [comment]
Yes, you are probably right.  People should expect behaviour analogous to gcc.  It is also useful for simpler Makefiles.

I'll implement this requested feature.

**Comment by vnmakarov** [comment]
I've fixed it by https://github.com/vnmakarov/mir/commit/053a1d41abe501dc258e185bbd2af4f4b734aab2

**Comment by edubart** [comment]
Thanks! This simplified my build scripts that uses C2M.
